packages/mobile/src/app/saga.ts

```diff
@@ -156,6 +156,8 @@ export function* handleDeepLink(action: OpenDeepLink) {
     } else if (rawParams.path.startsWith('/cash-in-success')) {
       // Some providers append transaction information to the redirect links so can't check for strict equality
       navigateHome()
+    } else if (rawParams.path.startsWith('/cash-in-failure')) {
```

Comment:
Should we show any error message in this case or suggest another provider?

Reply:
Yeah I think eventually having a separate landing experience is a good idea. This was a good reminder to route them back to the start of the flow, rather than to home.

Important context is that they will already get an explanatin from the provider as to why their transaction failed and then have to press a button to "go back to Valora". I don't think we can provide them with any more helpful information beyond that at this time.